languages/elixir/exercises/concept/binary-matching/.meta/example.ex

```diff
@@ -0,0 +1,24 @@
+defmodule FileSniffer do
+  def type_from_extension(<< "bmp" >>), do: {:ok, "image/bmp"}
+  def type_from_extension(<< "png" >>), do: {:ok, "image/png"}
+  def type_from_extension(<< "jpg" >>), do: {:ok, "image/jpg"}
+  def type_from_extension(<< "gif" >>), do: {:ok, "image/gif"}
+  def type_from_extension(<< "exe" >>), do: {:ok, "application/octet-stream"}
```

Comment:
Is there a reason to write `<< "exe" >>` instead of just `"exe"`? It's the same thing, right?

Reply:
It is the same.  There is no reason other than I think when I was tinkering with the exercise I probably had something else in the match.